web_client/src/components/ControlPanel.vue

```diff
@@ -206,52 +206,64 @@ export default {
             elevation="3"
           >
             <div class="d-flex">
-              <v-container>
-                <v-row no-gutters>
-                  <v-col cols="3">
+              <div
+                class="d-flex px-5 py-3"
+                style="flex-direction: column; flex-grow: 3"
```

Comment:
Do Vuetify's normal flex- and layout-related components and classes not handle what we want to do here?

Reply:
Changed in [ce6e59e](https://github.com/OpenImaging/miqa/pull/563/commits/ce6e59e61637edf4ce55cc41b430f93f70a9fc28)